.github/workflows/release.yml

```diff
@@ -35,19 +37,26 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup
-        uses: ./.github/actions/setup
+        id: config
+        uses: ./.github/actions/init
+        with:
+          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
+          turbo-team: ${{ vars.TURBO_TEAM }}
+          turbo-token: ${{ secrets.TURBO_TOKEN }}
+          playwright-enabled: true # Must be present to enable caching on branched workflows
 
       - name: Create Release PR
         id: changesets
         uses: changesets/action@v1
         with:
           commit: "chore(repo): Version packages"
-          publish: npm run release
+          publish: npx turbo build $TURBO_ARGS && changeset publish && git push --follow-tags
```

Comment:
@tmilewski `npm run release` performs a clean build and then publishes the packages. I think that we explicitly wanted to build the packages with force to avoid any cache-related issue. With this change it seems that we may use the cache in the release action. Are we sure about this change?

cc: @nikosdouvlis

Reply:
Yup, we can force elsewhere.